panels/logbook/ha-panel-logbook.html

```diff
@@ -106,10 +112,11 @@
       // ISO8601 formatted date string
       _currentDate: {
         type: String,
-        value: function () {
-          var value = new Date();
-          var today = new Date(Date.UTC(value.getFullYear(), value.getMonth(), value.getDate()));
-          return today.toISOString().split('T')[0];
+        value: () => {
```

Comment:
What's this change doing? It does not seem related to the refresh button nor is it a simple conversion of var => const

Reply:
TBH, I'm not completely sure, it was requested by @c727 above. Something to do with timezones apparently?

Reply:
toISOString() does not return the local date

Reply:
as far as I know, the backend parses the date as a UTC date. So that seems right?

Reply:
`_cuttentDate` is the date selected in the picker.
`_computeFilterDate(_cuttentDate)` is the ISO formatted string to request the data

Reply:
hmz nevermind... using the `Date.UTC()` aslo produces correct dates

Sorry^^